Library/Homebrew/utils/service.rb

```diff
@@ -0,0 +1,50 @@
+# typed: true
```

Comment:
If we teach these to brew, can you make a PR to switch brew services to these?

Reply:
Sure, I can see if that's possible though there isn't a ton of shared logic here. My radical idea is that sometime in the future we should probably merge `brew services` into the main brew repo since there's already a bunch of services specific code here anyway but if that ever happens, it's not going to happen anytime soon.

Reply:
> My radical idea is that sometime in the future we should probably merge `brew services` into the main brew repo since there's already a bunch of services specific code here anyway

This would make sense to me and would probably be a lot less hard than you'd think.